README.md

```diff
@@ -1,3 +1,6 @@
 # [The Algorithms](https://github.com/TheAlgorithms) old website
 
-This website is deprecated. The new website is <https://the-algorithms.com>. You can see the old website by going to <https://thealgorithms.github.io/index-old.html>.
+### This website is deprecated. 
+> The new website is <https://the-algorithms.com>. <br>
+
+> You can see the old website by going to <https://thealgorithms.github.io/index-old.html>.
```

Comment:
What do you think about making it look like this?
```suggestion
> The new website is <https://the-algorithms.com>.
> You can see the old website by going to <https://thealgorithms.github.io/index-old.html>.
```

Reply:
> What do you think about making it look like this?

Well, It looks more better like organized in the sense of points about old and new.